pages/index.vue

```diff
@@ -1,9 +1,53 @@
 <template>
-  <NuxtLink :to="`/usecase_0/panel_1`">
-    Example use case
-  </NuxtLink>
+  <div class="flex flex-col w-screen h-screen bg-gray-200 p-2">
+    <!-- Banner -->
+    <div class="banner">
+      <Banner />
+    </div>
+
+    <div class="flex overflow-auto">
+      <!-- usecases -->
+      <div class="m-2 bg-white rounded">
+        <!-- category 1 -->
+        <h3 class="text-2xl">
+          Category 1
+        </h3>
+        <div class="flex text-left">
+          <UseCaseCard
+            v-for="usecase in usecases.items"
+            :key="usecase.id"
+            :card="usecase"
+          />
+        </div>
+        <!-- category 2 -->
+        <h3 class="text-2xl">
+          Category 2
+        </h3>
+      </div>
+
+      <!-- Tags -->
+      <div class="m-2 bg-white rounded">
+        <h3>
+          Tags
```

Comment:
This now looks a bit strange, because we don't have tags (yet). Maybe we can add some dummy tags, like "CPM data", "spatial merging", and "constraints"?

Maybe add some styling as well, similar to e.g. https://cdn.business2community.com/wp-content/uploads/2014/01/tags.jpg

Reply:
Agree. But I think it is better that we leave it there as we will need it later. Just take it into consideration when doing styling. I will add some dummy tags in a separate PR.